### ISSUE TITLE
Not all image in the Software Manual page have a border

### ISSUE THREAD

**Issue by henkdegroot** [jamulussoftware/jamulussoftware.github.io#673]
**URL**
https://jamulus.io/wiki/Software-Manual

**Describe the bug**
Musician profile image and Input/Output channel mapping image do not have borders, while all other images (except the diagram) do have border applied.

It would be nice to have borders around these two images.

**Comment by gilgongo** [comment]
Yes, I thought maybe the smaller images didn't need borders perhaps. But maybe a thinner one would be nicer? BTW the purpose of the border is mainly so that white edges on screen crops don't blend in too much with the page (which come to think of it probably means those images need borders more than the full screen ones!)

**Comment by pljones** [comment]
@gilgongo is there any chance of this happening?

**Comment by pljones** [comment]
I'll move this to 3.9.1 now.

**Comment by pljones** [comment]
Leaving this in for now as I don't expect it to require translations.  If we get close to 3.9.1 and it looks like not getting done, it get move.

**Comment by pljones** [comment]
Moving to 3.10.0 as it's not progressing.

**Comment by pljones** [comment]
~Is anyone working on this for 3.10.0?  If not, I'll remove the milestone.~
No one assigned.  Moving to Triage and dropping the milestone.
